src/decimal.rs

```diff
@@ -72,6 +72,8 @@ impl Decimal {
     pub fn to_f64(&self) -> f64 {
         let (m, e) = self.decode();
         let mut d = m as f64;
+        // We probably don't want to call .cmp() because of performance loss
+        #[allow(clippy::comparison_chain)]
```

Comment:
Clippy wants IMO rewrite of `if-else` chain to `match`

Reply:
That's true, but the point of the comment is that we are probably aware of that and it's cheaper to do if-else than to call `.match()` as it may have performance penalty as stated at the warning description: https://rust-lang.github.io/rust-clippy/master/index.html#/comparison_chain